yt/odin/requirements.txt

```diff
@@ -3,4 +3,8 @@ Flask==2.0.3
 Werkzeug==2.2.2
 cheroot==10.0.0
 
+# Required by some of the checks
+requests==2.31.0
+tabulate==0.9.0
+
```

Comment:
You may also want to add `distro==1.9.0`, odin requires this package at the latest commit.